crates/astria-conductor/local.env.example

```diff
@@ -6,9 +6,15 @@
 # 12000 milliseconds is the default Celestia block time.
 ASTRIA_CONDUCTOR_CELESTIA_BLOCK_TIME_MS=12000
 
+# Disable using the auth header with celestia jsonrpc. Celestia nodes can be run
+# without authentication, in which case this should be set to true.
+ASTRIA_CONDUCTOR_NO_CELESTIA_BEARER_TOKEN=false
```

Comment:
Aiming for consistency here, ie `NO_METRICS`, where that could be inferred from not having a metrics url set. It might be good to explicitly disallow setting the token value to nil if `NO_CELESTIA_BEARER_TOKEN` is not true.

Reply:
synced offline, changed name to be more clear but keeping seperate config to keep in line with config standards